Makefile

```diff
@@ -49,23 +55,33 @@ HUB_PYTEST_DIR := $(HUB_REPORT_DIR)/pytest
 #Specifies that those commands don't create files
 .PHONY: clean clean_hub clean_sensor all sensor hub help flash_sensor sensor_cppcheck sensor_cppcheck_no_report sensor_clang_analyzer check_sensor eslint prospector pip pip-prospector python npm cpplint pip-cpplint gcov init prettier pip-gcovr
 
+###
 .DEAFULT: help
 help:
 	@echo "The following targets are available:"
-	@echo "sensor\t\t\truns the manufacturer provided makefiles \"all\" target"
-	@echo "flash_sensor\t\truns the manufacturer provided makefiles \"flash\" target"
-	@echo "\tTo run other targets from the manufacturer provided makefile, run \"$(SENSOR_MAKE) help\" to see what targets it has, and then run \"$(SENSOR_MAKE) <target>\" to run the desired target."
-	@echo "\n---------------------------------------------\nTODO list targets and what they do here\n---------------------------------------------\n"
+ 	@echo "sensor\t\t\truns the manufacturer provided makefiles \"all\" target"
+ 	@echo "flash_sensor\t\truns the manufacturer provided makefiles \"flash\" target"
+ 	@echo "\tTo run other targets from the manufacturer provided makefile, run \"$(SENSOR_MAKE) help\" to see what targets it has, and then run \"$(SENSOR_MAKE) <target>\" to run the desired target."
+ 	@echo "\n---------------------------------------------\nTODO list targets and what they do here\n---------------------------------------------\n"
+
 	@echo "make prepare-dev"
-	@echo "	       prepare development environment, use only once"
-	@echo "make test"
+	@echo "	      prepare development environment, use only once"
+	@echo "make test:
 	@echo "       run tests"
-	@echo "make lint"
-	@echo "	      run pylint and mypy"
 	@echo "make run"
 	@echo "	      run project"
-	@echo "make doc"
-	@echo "	      build sphinx documentation"
+
+make prepare-dev:
```

Comment:
make prepare-dev:
 	sudo apt-get -y install python3.5 python3-pip
 	python3 -m pip install virtualenv
 	make venv
These should be taken care of with the init target which runs the init script. Was there a reason behind adding these? Are we sure we want to use python 3.5 specifically? Where's the venv target?

Reply:
You can change the 3.5 to 3. or if anything take these help targets out